src/Tasks/Monitor/HealthChecks/MaximumStorageInMegabytes.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Spatie\Backup\Tasks\Monitor\HealthChecks;
+
+use Spatie\Backup\Helpers\Format;
+use Spatie\Backup\Tasks\Monitor\HealthCheck;
+use Spatie\Backup\BackupDestination\BackupDestination;
+
+class MaximumStorageInMegabytes extends HealthCheck
+{
+    protected $allowance;
```

Comment:
Let's rename this to something more meaningful like `maximumSizeInMegaBytes`. I don't care that it's a long variable name.